Dockerfile

```diff
@@ -67,9 +65,10 @@ RUN set -ex\
 	;
 
 # Build-static downloads the static javascript.
-FROM build-python AS build-static
-COPY --chown=0:0 static/  ./static/
-COPY --chown=0:0 *.json *.js  ./
+FROM registry.access.redhat.com/ubi8/nodejs-10 AS build-static
+WORKDIR /opt/app-root/src
+COPY --chown=1001:0 static/  ./static/
+COPY --chown=1001:0 *.json *.js  ./
```

Comment:
see https://catalog.redhat.com/software/containers/ubi8/nodejs-10/5c839aa3d70cc51dd4c425d9?container-tabs=dockerfile for background on ownership change